sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/tasks/TaskManagementSteps.java

```diff
@@ -53,5 +76,152 @@ public TaskManagementSteps(WebDriverHelpers webDriverHelpers) {
           webDriverHelpers.fillAndSubmitInWebElement(
               GENERAL_SEARCH_INPUT, EditCaseSteps.aCase.getUuid());
         });
+
+    When(
+        "^I search last created task by API using Contact UUID and wait for (\\d+) results to be displayed$",
+        (Integer displayedResults) -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(GENERAL_SEARCH_INPUT);
+          webDriverHelpers.fillAndSubmitInWebElement(
+              GENERAL_SEARCH_INPUT, apiState.getCreatedContact().getUuid());
+          webDriverHelpers.waitUntilNumberOfElementsIsReduceToGiven(TABLE_ROW, displayedResults);
+        });
+
+    When(
+        "^I am checking if all the fields are correctly displayed in the Task Management table$",
+        () -> {
+          org.sormas.e2etests.pojo.api.Task expectedTask = apiState.getCreatedTask();
```

Comment:
remove import

Reply:
This import is needed because we already have a Task, which is from web, and this one is from api.

Reply:
got it